### ISSUE TITLE
Rubocop show me an offenses on CI but not locally

### ISSUE THREAD

**Issue by benoittgt** [bbatsov/rubocop#4657]
## Expected behavior

Don't display offense on CI when the offense is not visible locally.

## Actual behavior

I have removed [the empty line](https://github.com/codetriage/codetriage/pull/610/files#diff-1acd2e7e27a227829d5d14a91c863bb6L14) but I still get [an offense](https://travis-ci.org/codetriage/codetriage/jobs/263391897)

```sh
9.49s$ rubocop --rails --display-cop-names --cache false
Inspecting 207 files
...............................................................................................................................................................C...............................................
Offenses:
db/schema.rb:14:1: C: Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body beginning.
207 files inspected, 1 offense detected
The command "rubocop --rails --display-cop-names --cache false" exited with 1.
```

## Steps to reproduce the problem

Sadly none. I can't reproduce the issue locally. 

## RuboCop version

```
$ rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin16)
```

**Comment by benoittgt** [comment]
That's related to my migrate script.
